pulsar-io/kafka/src/main/java/org/apache/pulsar/io/kafka/KafkaAbstractSink.java

```diff
@@ -49,7 +53,11 @@
     @Override
     public void write(Record<byte[]> sourceRecord) {
         KeyValue<K, V> keyValue = extractKeyValue(sourceRecord);
-        ProducerRecord<K, V> record = new ProducerRecord<>(kafkaSinkConfig.getTopic(), keyValue.getKey(), keyValue.getValue());
+        KeyValue<Schema, Schema> keyValueSchemas = extractKeyValueSchemas(sourceRecord);
```

Comment:
Probably using a KeyValue<Schema, Schema> here may be misleading, what about using a "Pair" ?

I mean, that usually KeyValue is the class we use ad content of the Record, here you are using it as a simple Pair, to return two objects from the extractKeyValueSchemas method